crates/topos-p2p/src/runtime/mod.rs

```diff
@@ -62,16 +54,24 @@ impl Runtime {
 
         self.bootstrapped = true;
 
-        self.swarm.add_external_address(self.addresses.clone());
+        debug!("Added external addresses: {:?}", self.public_addresses);
```

Comment:
```suggestion
        debug!("Added public addresses: {:?}", self.public_addresses);
```

not sure as I see that _public_ addresses are _external_ addresses for the swarm